chart/values.yaml

```diff
@@ -387,7 +387,7 @@ components:
     remoteStorage:
       kind: minio
       minio:
-        endpoint: minio:9000
+        endpoint: :9000
```

Comment:
This could be `minio`, just to make the endpoint more explicit. It should work without `namespace.svc.cluster.local`, too.